install/Knative-with-IKS.md

```diff
@@ -135,8 +135,14 @@ components, the recommended configuration for a cluster is:
 
 Knative depends on Istio.
 
-1.  Install Istio:
+1.  Install Istio.
+    Note, we install just the Istio CRDs first and give Kubernetes a chance
+	to create those resource definitions. Sometimes the deployment of
+	`istio.yaml` (which includes those CRD definitions) fails due to the CRD
+	controller not defining those types by the time we try to use them:
     ```bash
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/istio-crds.yaml
```

Comment:
IMO at this point we should list this command like:

```
1 && \
  2 && \
  3
```

so that if people copy-paste all of this code block, it stops on failure.

Also I wonder if you still see errors when you don't add the `sleep 10` (I feel like applying CRDs first is already a big improvement on the success rate).

Reply:
Yep - I mentioned in the initial comment that once we get the text right, and people are ok with it, I can make the same change to the others.  
Will add the &&'s